.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+  - "6"
+  - "5"
+  - "4.4.2"
+
+services:
+  - docker
+  - redis
+
+before_install:
+  - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
+  - npm cache clean
+  - docker build --tag travis .
+  - docker run -d --net=host -e NODE_ENV=ci -e DISABLE_CSP=true travis
+
+script:
+  - npm test && npm run test:acceptance
```

Comment:
do you need the npm run test:acceptance since the npm test runs the acceptance anyway

https://github.com/UKHomeOffice/right-to-rent-check/blob/master/package.json#L11